extension/scripts/features/travelitemprofits/ttTravelItemProfits.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Travel Item Profits",
+		"travel",
+		() => settings.pages.travel.travelProfits,
+		null,
+		addProfitsColumn,
+		removeProfitsColumn,
+		{
+			storage: ["settings.pages.travel.travelProfits"],
+		},
+		() => {
+			if (!hasAPIData()) return "No API access.";
+		}
+	);
+
+	async function addProfitsColumn() {
+		await requireElement(".travel-agency-market .items-list-title");
+		document.documentElement.classList.add("tt-travel-profits");
+		if (document.find(".tt-travel-market-heading")) return;
+		const market = document.find(".travel-agency-market");
+		const headings = market.find(".items-list-title");
+
+		const profitHeading = document.newElement({ type: "div", text: "Profit", class: `tt-travel-market-heading tt-title-${settings.themes.containers}` });
+		headings.insertBefore(profitHeading, headings.find(".stock-b"));
+
+		const rows = document.findAll(".users-list > li");
+		for (let row of rows) {
+			const id = parseInt(row.find(".details").getAttribute("itemid"));
+			const marketPrice = parseInt(torndata.items[id].market_value);
+			const buyPrice = parseInt(row.find(".cost .c-price").innerText.replace(/[$,]/g, ""));
+			const profit = marketPrice - buyPrice;
+
+			const span = document.newElement({ type: "span", class: "tt-travel-market-cell", attributes: { value: profit } });
+			const innerSpan = document.newElement({ type: "span", text: `${profit < 0 ? "-$" : "+$"}${formatNumber(Math.abs(profit))}` });
+
+			if (buyPrice > marketPrice) span.style.color = "#de0000";
```

Comment:
I'll have to test these colors whether they can pass for both light and dark mode.